Extensions/Effects/glitch-pixi-filter.js

```diff
@@ -1,13 +1,16 @@
 gdjs.PixiFiltersTools.registerFilterCreator('Glitch', {
   makePIXIFilter: function(layer, effectData) {
     var glitchFilter = new PIXI.filters.GlitchFilter();
-
+    glitchFilter._animationTimer = 0;
     return glitchFilter;
   },
   update: function(filter, layer) {
-    if (filter.animated) {
-      filter.time += layer.getElapsedTime() / 1000;
-      filter.seed = Math.random();
+    if (filter.animationFrequency !== 0) { 
+      filter._animationTimer += layer.getElapsedTime() / 1000;
+      if (filter._animationTimer >= 1 / filter.animationFrequency) {
+        filter.seed = Math.random();
```

Comment:
If my seed affected by speed formula is ok by @4ian, please simplify this logic to how it was before and use it here too